### PR TITLE
fix: remove console.error to fix crash on iOS

### PR DESCRIPTION
**Pull request by dominictb**
### Related Issues

https://github.com/Expensify/App/issues/45623

### Manual Tests

1. Run example iOS app
2. Input `[ a ](https://example.com)`

Verify that there's no crash, and the markdown input still applies the style for the text

### Linked PRs

https://github.com/Expensify/expensify-common/pull/773  - Hold until this is merged
Pending: create bump version PR in app repo.

### Evidence

md-ios

### CODE REVIEW

**Comment by tomekzaw** [comment]
Thanks for the PR! I will merge this PR once https://github.com/Expensify/expensify-common/pull/773 is merged.

File: parser/react-native-live-markdown-parser.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
...g.number.national=p2(t2,this.l,2),this.g.number.e164=p2(t2,this.l,0),this.g.number.rfc3966=p2(t2,this.l,3),this.g.number.significant=c2(this.l),this.g.canBeInternationallyDialled=t2.ja(this.l),this.g.valid=O1(this.l),this.g.possible=y1(this.l),this.g.possibility=W1(this.l),!this.g.valid&&this.g.possible&&(this.g.possibility="invalid",this.g.possible=!1),this.g.type=Z0(this.l),this.g.typeIsMobile=this.sa(),this.g.typeIsFixedLine=this.ra(),this.g.countryCode=h2(i)?N2(t2,i):0}var R2=["PhoneNumber$$module$src$index"],s2=v;R2[0]in s2||typeof s2.execScript>"u"||s2.execScript("var "+R2[0]);for(var E2;R2.length&&(E2=R2.shift());)R2.length||L===void 0?s2[E2]&&s2[E2]!==Object.prototype[E2]?s2=s2[E2]:s2=s2[E2]={}:s2[E2]=L;function s1(u){return h2(u)?N2(t2,u):0}L.getCountryCodeForRegionCode=s1,L.getRegionCodeForCountryCode=function(u){return A2(u)};function j1(u){var d={};return u.filter(function(t){return d.hasOwnProperty(t)?!1:(d[t]=1,!0)})}L.getSupportedRegionCodes=function(){return j1(X0())},L.getSupportedCallingCodes=function(){return j1(K0())},L.getExample=function(u,d){var t;return d?t=T1(u,J0(d)):t=T1(u,0),new L(t,u).toJSON()},L.getAsYouType=function(u){return new u2(u)},L.getNumberFro
```
Comment (DylanDylann) [inline parser/react-native-live-markdown-parser.js:3]: Oops, is this change needed?

**Review by tomekzaw — APPROVED** [review]
Looks good to me
